Docs 5955 mq address mq 1715 and mq 1717 (#45)
mulesoft/docs-mq: modules/ROOT/pages/mq-understanding.adoc

* *Message TTL*
+
Each queue has a message time-to-live (TTL). TTL specifies how long the message is in the queue before Anypoint MQ expires the message and removes it from the queue.
[message_lock_ttl]
* *Message Lock Default TTL* 

> How long a message waits when the message is not acknowledged before being put back into the queue, for example, in the event of server failure or when flow processing takes too long and times out.

Reviewer: ```suggestion
How long a message remains unacknowledged before being returned to the queue: for example, in the event of server failure or when flow processing takes too long and times out.
```
